daemon/daemon.go

```diff
@@ -69,7 +77,7 @@ type Daemon struct {
 	standbyOpinions *standby.StandbyOpinions
 
 	// set to remember we need to restart the system
```

Comment:
the comment should probably be changed to something like:

// set to what kind of restart was requested if any